CONTRIBUTING.md

```diff
@@ -21,24 +21,24 @@ Before doing anything else, run these commands:
 ```
 git clone git@github.com:chakra-ui/chakra-ui.git
 cd chakra-ui
-yarn install
-yarn bootstrap
-yarn build
+yarn run install
```

Comment:
I don't believe that `yarn run install` is valid. Believe this should be `yarn install`